src/manage-custom-commands.tsx

```diff
@@ -36,19 +39,26 @@ export default function Command() {
 
   React.useEffect(() => {
     (async () => {
+      const allCommandsRaw = await LocalStorage.allItems();
+
+      const allCommands = Object.keys(allCommandsRaw).reduce<CommandRecord[]>((commands, id) => {
+        commands.push(JSON.parse(allCommandsRaw[id]));
+        return commands;
+      }, []);
+
       const frontmostApplication = await getFrontmostApplication();
 
       const { name: frontmostApplicationName } = frontmostApplication;
       const isInBrowserNow = isCurrentAppBrowser({
         frontmostApplicationName,
         preferenceBrowserName: browser?.name || "",
       });
-      let activeTabUrl = null;
+      let activeTabUrl: State["activeTabUrl"] = null;
       if (isInBrowserNow) {
         activeTabUrl = await getActiveTabUrl(frontmostApplicationName);
       }
 
-      const storedData = [applicationShortcutRaiseHandForMeet, applicationShortcutRecordToggleMute];
+      const storedData = [applicationShortcutRaiseHandForMeet, applicationShortcutRecordToggleMute, ...allCommands];
```

Comment:
I'm thinking what should we do with pre-defined shortcuts. Maybe we should do nothing but let's add on edit screen that this is not editable and any changes will be reverted to defaults

Reply:
1. There is no ability to mix Form and Detail view, but I add confirmation alert for this. 
2. Maybe we should add ability to delete theme. Just add optional field to predefined commands `isDeleted`